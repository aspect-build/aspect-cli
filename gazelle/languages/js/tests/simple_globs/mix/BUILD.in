# gazelle:js_files **/*.ts
# gazelle:js_test_files **/*.spec.ts
# gazelle:js_test_files **/*.mock.ts
