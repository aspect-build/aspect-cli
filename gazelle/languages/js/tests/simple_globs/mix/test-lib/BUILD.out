load("@aspect_rules_ts//ts:defs.bzl", "ts_project")

# Output everything as a standard lib, no test target

# Only contains a standard target, no tests
# gazelle:js_test_files there-are-no-tests

ts_project(
    name = "test-lib",
    srcs = [
        "faker.ts",
        "normally.mock.ts",
        "normally.spec.ts",
        "normally.test.ts",
    ],
)
